build_tools/github_actions/amdgpu_family_matrix_xfail.py

```diff
@@ -11,6 +11,14 @@
             "expect_failure": True,
         }
     },
+    "gfx950": {
```

Comment:
Can this be tested on the PR itself using a github label to opt-in to this build? Or are you testing with workflow_dispatch somewhere?

Reply:
I raised this in internal chat as I am not sure but wanted to send this for early feedback.